OpenRA.Mods.Common/UtilityCommands/ExtractSettingsDocsCommand.cs

```diff
@@ -59,7 +59,7 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			foreach (var section in sections.OrderBy(s => s.Key))
 			{
 				var fields = section.Value.GetType().GetFields();
-				if (fields.Length > 0 && fields.Where(field => field.GetCustomAttributes<DescAttribute>(false).Length > 0).Count() > 0)
+				if (fields.Length > 0 && fields.Any(field => field.GetCustomAttributes<DescAttribute>(false).Length > 0))
```

Comment:
We can simplify it even further

```suggestion
				if (fields.Any(field => field.GetCustomAttributes<DescAttribute>(false).Length > 0))
```